src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesV2Spec.scala

```diff
@@ -411,6 +411,43 @@ class ResourceRoutesV2Spec extends AnyFlatSpec with Matchers with TestSupport wi
     }
   }
 
+  it should "204 deleting a child resource" in {
```

Comment:
this test was moved from below and amended not to require the `remove_child` action

Reply:
thanks for these comments!